lib/gettext-setup/pot.rb

```diff
@@ -50,7 +50,7 @@ def self.string_changes?(old_pot, new_pot)
       return true
     end
 
-    def self.generate_new_pot(locales_path = GettextSetup.locales_path, path = nil)
+    def self.generate_new_pot(path = nil, locales_path = GettextSetup.locales_path)
```

Comment:
Why are we changing the order of these? They're currently consistent with `update_pot`.

Reply:
I'm fine with switching (omitting the second argument in later calls makes sense), I think it's worth keeping these in sync though.